src/wazuh_modules/syscollector/CMakeLists.txt

```diff
@@ -95,6 +98,37 @@ elseif(UNIX AND NOT APPLE)
   endif(NOT CMAKE_SYSTEM_NAME STREQUAL "AIX")
 endif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
 
+if (NOT ${TARGET} STREQUAL "")
```

Comment:
Maybe this conditional is unnecessary.
Why not directly ask if ${TARGET} is equal to a specific string?

Reply:
If the target is not declared it fails. Also included here https://github.com/wazuh/wazuh/pull/18199